cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -152,12 +201,13 @@ SHOW BTREE INDEXES
 a|
 B-tree indexes are deprecated and need to be migrated before upgrading to 5.0.
```

Comment:
Unrelated to this PR, but I think we might wanna change this wording. Maybe we should do something like 'B-tree indexes are deprecated and will be replaced in 5.0 by `<link-to-future-indexes>`', and maybe something about migration but we don't know what that will look like yet..

Reply:
Yes, when we have decided on the wording we like, I think it would be nice to change the warning message as well